src/entities/communitySbt.ts

```diff
@@ -635,39 +646,34 @@ class SBT {
    * the awarded time as end of season and claimed time
    * as eithr zero if not claimed or subgrap's minted timestamp
    */
-  async constructTopBadge(
+  static async constructTopBadge(
```

Comment:
what does static mean in this context?

Reply:
static is used when the function doesn't depend on the object itself (i.e. it doesn't use any fields of ``this``)